hazelcast/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlanBuilder.java

```diff
@@ -373,7 +373,7 @@ public static Map<MemberInfo, int[]> getPartitionAssignment(
 
             // Interactive prunable queries may require coordinator to be present
             // If coordinator still not captured to participate in the job -- do it.
-            extraRequiredMemberAddresses.forEach(requiredMemberAddr -> {
+            extraRequiredMemberAddresses.forEach(requiredMemberAddr -> { // NOSONAR : already null-checked.
```

Comment:
maybe you could make `@Nullable Set<@Nonnull Address> extraRequiredMemberAddresses`. Does sonar complain about nullability of the collection or the elements?
Alternatively use something like `extraRequiredMemberAddresses = checkNotNull(extraRequiredMemberAddresses);` + mark `checkNotNull` method as `@Nonnull`